src/analytics/private/athena-saved-queries.ts

```diff
@@ -34,17 +33,19 @@ export class AthenaSavedQuery extends Construct {
     super(scope, id);
 
     const mustacheParam: MustacheParamType = {
```

Comment:
how about cast the props as `MustacheParamType`

Reply:
fixed